path_2024.py

```diff
@@ -32,7 +33,7 @@ def run() -> int:
     Reads in hardcoded waypoints from CSV file and sends drone commands.
     """
     # Wait ready is false as the drone may be on the ground
-    drone = dronekit.connect(CONNECTION_ADDRESS, wait_ready = False)
+    drone = dronekit.connect(CONNECTION_ADDRESS, wait_ready = False, timeout = TIMEOUT)
```

Comment:
If we set `wait_ready` to False than the timeout here doesn't do anything. but yea it would help if we want to set `wait_ready` back to True in the future.